Packs/F5/Integrations/F5/integration-F5.yml

```diff
@@ -130,19 +158,61 @@ script:
             }
             return 'Gevald!!';
         default:
-            return sendRequest(getMethod(command), getUrl(command, args), buildBody(command, args));
+            body = sendRequest(getMethod(command), getUrl(command, args), buildBody(command, args));
+            md = tableToMarkdown("F5 Results", body);
+            cmd_arr = command.split('-');
+            key = "F5." + cmd_arr[cmd_arr.length - 1];
+            entry = {
+                'Type' : entryTypes.note,
+                'Contents': body,
+                'ContentsFormat' : formats.json,
+                'HumanReadable': md,
+                'ReadableContentsFormat' : formats.markdown
+            };
+            entry['EntryContext'] = {}
+            entry['EntryContext'][key] = body
+
+            return entry;
     }
   type: javascript
   commands:
   - name: f5-create-policy
-    arguments: []
+    arguments:
+    - name: name
+      required: true
+      description: The name of the policy to create
+    - name: description
+      required: true
+      description: The description of the policy
+    - name: copyFrom
```

Comment:
These parameters need to match the API documentation: https://clouddocs.f5.com/api/icontrol-rest/APIRef_tm_security_firewall_policy.html since this integration handles passing the API request body arguments by converting the command arguments into a JSON body without modification. Can we consider this an improvement for phase II refactor?

Reply:
ok